cluster/gce/trusty/helper.sh

```diff
@@ -18,17 +18,45 @@
 
 # The configuration is based on upstart, which is in Ubuntu up to 14.04 LTS (Trusty).
 # Ubuntu 15.04 and above replaced upstart with systemd as the init system.
-# Consequently, the configuration cannot work on these images.
+# Consequently, the configuration cannot work on these images. In release-1.2 branch,
+# GCI and Trusty share the configuration code. We have to keep the GCI specific code
```

Comment:
I am preparing for a cluster/gce/gci dir which serves gci only. But such a big change will be in 1.3 only. For 1.2.X gci will still use the code in cluster/gce/trusty. Later if a PR breaking GCI is cherry picked to release-1.2, we will have to make 3 PR: one for fixing master branch cluster/gce/gci, one for fixing master branch cluster/gce/trusty, and another for cherry-pick into cluster/gce/trusty in release-1.2. If we only keep gci-specific code in master branch, it is hard to cherry-pick into release-1.2 as there is no dir cluster/gce/gci